Tests/test_file_png.py

```diff
@@ -620,16 +613,536 @@ def test_exif_argument(self):
         with Image.open(test_file) as reloaded:
             self.assertEqual(reloaded.info["exif"], b"Exif\x00\x00exifstring")
 
-    @unittest.skipUnless(
-        HAVE_WEBP and _webp.HAVE_WEBPANIM, "WebP support not installed with animation"
-    )
-    def test_apng(self):
-        with Image.open("Tests/images/iss634.apng") as im:
+    # APNG browser support tests and fixtures via:
+    # https://philip.html5.org/tests/apng/tests.html
+    # (referenced from https://wiki.mozilla.org/APNG_Specification)
+    def test_apng_basic(self):
+        im = Image.open("Tests/images/apng/single_frame.png")
```

Comment:
Use context manager for these? cf. https://github.com/python-pillow/Pillow/pull/4232

Reply:
Fixed in e7f47d2